kategory/src/test/kotlin/kategory/data/EvalTest.kt

```diff
@@ -8,7 +8,10 @@ import org.junit.runner.RunWith
 class EvalTest : UnitSpec() {
     init {
 
-        testLaws(MonadLaws.laws(Eval, EvalEq()))
+        testLaws(MonadLaws.laws(Eval, object : Eq<HK<Eval.F, Int>> {
```

Comment:
Gotcha! Also, woops :P Could you please make a generic version instead that wraps another Eq?